client/components/pages/AuthPage/AuthPage.js

```diff
@@ -0,0 +1,106 @@
+import React, { Component } from 'react'
+import { Link, Redirect } from "react-router-dom";
+
+export const LOGIN = 'login';
+export const REGISTER = 'register';
+export const FORGOT = 'forgot';
+export const RESET = 'reset';
+
+const ResetPage = ({ isActive }) => isActive && (
+    <>
+        <form>
+            <p className="title" >Reset Password</p>
+            <input type="Password" placeholder="Password" className="inputField" />
+            <input type="Password" placeholder="Re-Enter Password" className="inputField" />
+            <button type="submit">Reset</button>
+        </form>
+        <div className="backgroundImage" />
+    </>
+)
+
+const ForgotPage = ({ isActive }) => isActive && (
+    <>
+        <form>
+            <p className="title">Forgot Password</p>
```

Comment:
Move the title and description into a separate component and use that instead since this seems repetitive